intro_to_python/intro_to_python.md

```diff
@@ -188,84 +196,88 @@ If you are interested in using Python for data analysis, you might want to write
 
 - You can intersperse sections of text, written in Markdown language, that can provide explanations and context for your code. This is especially valuable for data analysis, similar to a scientific notebook.
 
-Interested in learning how to write Python code in a Jupyter notebook? Continue on to the next section for a hands-on activity! You won't need to download anything to your computer.
-
-
-## Using a Jupyter Notebook
-
-This is a hands-on activity! Feel free to pop over to that tab you opened in the [Preparation section](#Preparation) to get started. Need the link again? Here it is:
-
-<a href = "https://mybinder.org/v2/gh/arcus/education_modules/main?labpath=intro_to_python%2Fnotebooks%2Fintro-to-python.ipynb" target = "_blank"> <img src = "https://mybinder.org/badge_logo.svg"></a> **← Click the "launch binder" button!**
-
-### Knowledge Check 2
-
-Which of the following is **not** an example of Python code?
-
-[( )] `print("Python is the best!")`
-[(X)] `print Hello World!`
-[( )] `list(range(1, 100))`
-[( )] `height = input("How tall are you?")`
-***********************************************************************
-<div class = "answer">
-Python uses functions that contain instructions that tell the computer what it is supposed to do, and these functions are the name of the function followed by the **arguments** in parentheses. Arguments are information that the person writing the code (you!) provide to tell the function what data to perform its task on and more specificity on how exactly to perform the task. `print()`, `list()`, `range()`, and `input()` are just a few of Python's built-in functions! [You can read more about Python's built-in functions here](https://docs.python.org/3/library/functions.html).
-</div>
-***********************************************************************
-
 
-### Knowledge Check 3
+### Quiz: Jupyter Notebooks
 
 What are the components of a Jupyter notebook? Select all that apply.
 
 [[X]] Code cells
 [[X]] Text cells written in Markdown
 [[ ]] A console where you can write lines of code interactively
-***********************************************************************
+***
 <div class = "answer">
 Jupyter notebooks contain code cells and text cells. The code cells are written in Python and can contain as much or as little code as you want; the output of the code can be displayed directly beneath the code cell and the code can be edited and re-run at any time. The text cells are written in Markdown language and can provide more context and reasoning than can easily be done using comments. Additionally, using Markdown allows you to add useful formatting to your text. Jupyter notebooks do not have console, but since you can run each code cell independently, you don't really need one!
 </div>
-***********************************************************************
+***
 
-
-## Next Steps
+## Accessing Python on Your Computer
```

Comment:
```suggestion
## Accessing Python on Your Computer or in the Cloud
```

```suggestion
## Accessing Python
```
I think either of these would work -- just don't want someone to glance at the TOC and think "on your computer" means running locally and then decide this module isn't right fit because they're trying to learn how to do in the cloud/google colab stuff.